cmd/src/server.rs

```diff
@@ -272,12 +272,14 @@ impl TiKVServer {
         &mut self,
         gc_worker: &GcWorker<RaftKv<ServerRaftStoreRouter>>,
     ) -> Arc<ServerConfig> {
+        let mut cfg_controller = ConfigController::new(self.config.clone());
         // Create CoprocessorHost.
         let mut coprocessor_host =
             CoprocessorHost::new(self.config.coprocessor.clone(), self.router.clone());
 
         let lock_mgr = if self.config.pessimistic_txn.enabled {
             let lock_mgr = LockManager::new();
+            cfg_controller.register("pessimistic_txn", Box::new(lock_mgr.config_manager()));
```

Comment:
`pessimistic_txn` or `pessimistic-txn`?

Reply:
`pessimistic_txn` same as the `TiKvConfig` field name.